sema/ro/creator/roblueprint.py

```
@@ -0,0 +1,63 @@
+        }
+
+        self.version = version
+        if not context:
+            self.context = context_uris[version]
+        else:
+            self.context = context
+        
+        self.body = body or {}
+        self.nested_datasets = nested_datasets
+        self.implicit_body = {k: v for k, v in self.body.items() if "*" in k}
+        self.explicit_body = {k: v for k, v in self.body.items() if not ("*" in k)}
```

Ruff (0.12.2)

Test for membership should be `not in`

Convert to `not in`

(E713)